server/src/main/java/org/opensearch/search/profile/ProfileResult.java

```diff
@@ -186,6 +260,22 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         return builder.endObject();
     }
 
+    private void createBreakownView(XContentBuilder builder) throws IOException {
```

Comment:
please correct the name createBreak **d** ownView. d is missing

Reply:
Resolved. Thank you.